src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletServiceImpl.java

```diff
@@ -82,7 +83,13 @@ public WalletDTO findById(Long id) {
     }
 
     @Override
-    public List<WalletDTO> findByName(String name) {
-        return getAll().stream().filter(walletDTO -> walletDTO.name().contains(name)).toList();
+    public List<WalletDTO> findAllByName(String name) {
+        List<WalletDTO> walletsDTO;
```

Comment:
I think that listOfWalletDTO would be more descriptive.

Reply:
In line 57 (methid getAll()) is same